src/Components/CommentSection.jsx

```diff
@@ -12,14 +12,18 @@ const CommentSection = ({id}) => {
         .then((res) => {
             setArticleComments(res)
             setLoading(false)
+        }).catch((er) => {
+            console.log(er)
+            setArticleComments([])
```

Comment:
This setting the comments to an empty array shouldn't be necessary. Your API should return an empty array if there are no comments for a particular article. This is a work around. 😄  Don't worry about fixing this on your backend project right now though but just understand it is not quite right. 👍